atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -147,7 +147,7 @@ def get_next_release(self, build_info):
         return next_release
 
     def reserve_build_in_koji(self, component, version, release, release_label,
-                              dockerfile_labels, source_build=False):
```

Comment:
can we just use `release` param, a.k.a if define always use it?

Reply:
in next_release_general is dockerfile updted with new release, so I need it

Reply:
ok